src/formatter.js

```diff
@@ -119,7 +120,11 @@ export default class Formatter {
   }
 
   wrapAsIdentifier(value) {
-    return this.client.wrapIdentifier((value || '').trim());
+    let hookContext;
+    if (this.builder && isFunction(this.builder.hookContext)) {
```

Comment:
@elhigu I'm a bit unsure about these checks (along with [this one](https://github.com/tgriesser/knex/pull/2314/files#diff-b27035181a616c40188ad22a32280732R136)).. they seem a bit hacky but I had to add them for CI to pass. do you know which cases formatter will be instantiated without a builder?